blog/_posts/2015-07-23-rubyherzlein.md

```diff
@@ -21,7 +21,7 @@ Why Rubyherzlein
 While deciding on a name, we talked about how the Rails Girls Society is so full of hearts! Its the logo for most events and also quite often used as an emoji within the RGSoC community. As we are beginners, we thought of calling ourselves Rubyherzlein, which in German means ‘little Ruby heart’! <3<3<3
 
 <img src="/img/blog/2015/rubyherzlein.JPG" alt="Team Rubyherzlein" width="600">
-<br><font color="grey"><small><i>Rubyherzlein Team + extended Team: Erik, Duana, Hannes  (Image: Rubyherzlein Team)</i></small></font> 
+<div align="center" class="image-credits">Rubyherzlein Team + extended Team: Erik, Duana, Hannes  (Image: Rubyherzlein Team) </div>
```

Comment:
could you remove the `align="center"` bit from all the files? Unfortunately some of the last contributors to blog posts seem to have added that, but it doesn't belong in there. Thank you so much!